modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,10 +5,109 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-665]
+== Release 6.6.5 (January 2022)
+
+Couchbase Server 6.6.5 was released in January 2022.
+This maintenance release includes bug fixes in multiple components.
+
+Additionally, this release updates _Log4J_ to version 2.17.1.
+
+[#fixed-issues-665]
+=== Fixed Issues
+
+This section highlights the notable issues fixed in this release.
+
+[#cluster-manager]
+==== Cluster Manager
+
+[#table_fixedissues_v665-cluster-manager,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47613[MB-47613^]
+| *Summary*: Add bucket names and UUIDs to pools/default
+
+| https://issues.couchbase.com/browse/MB-49271[MB-49271^]
+| *Summary*: ns_server should generate DCP connection names that are no longer than allowed by memcached
+
+|===
+
+[#data-service]
+==== Data Service
+
+[#table_fixedissues_v665-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-34280[MB-34280^]
+| *Summary*: Memcached should disallow DCP connections with names greater than 200 characters
+
+| https://issues.couchbase.com/browse/MB-49321[MB-49321^]
```

Comment:
Mention of following defect is good - https://issues.couchbase.com/browse/MB-49323

Reply:
I have added.